workalendar/tests/test_registry_africa.py

```diff
@@ -23,3 +25,13 @@ def test_africa(self):
         self.assertIn(Madagascar, classes)
         self.assertIn(SaoTomeAndPrincipe, classes)
         self.assertIn(SouthAfrica, classes)
+
+
+class SouthAfricaTest(GenericCalendarTest):
+    cal_class = SouthAfrica
+
+    def test_south_africa_2019(self):
+        holidays = self.cal.holidays_set(2019)
+
+        # variable days
+        self.assertIn(date(2019, 5, 8), holidays)
```

Comment:
looks fine, but I think I'd add a test in another year, to make sure that this date is NOT added to the holidays except in 2019.